pyproject.toml

```diff
@@ -55,6 +55,11 @@ Werkzeug = "^0.15.5"
 [tool.poetry.extras]
 markdown = ["markdown"]
 
+[tool.autopub]
+project-name = "more-categories"
```

Comment:
This field is intended for humans instead of computers, in places like the release page (see the ["Render Math 1.0.0"](https://github.com/pelican-plugins/render-math/releases) header as an example), so I would recommend using the following instead:
```
project-name = "More Categories"
```